Libs/DICOM/Widgets/ctkDICOMPatientItemWidget.h

```diff
@@ -155,16 +163,28 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMPatientItemWidget : public QWidget
   /// Add/Remove study item widgets
   Q_INVOKABLE void addStudyItemWidget(const QString& studyItem);
   Q_INVOKABLE void removeStudyItemWidget(const QString& studyItem);
+  Q_INVOKABLE ctkDICOMStudyItemWidget* geStudyItemWidgetByStudyItem(const QString& studyItem);
   ///@}
 
   /// Set selection for all studies/series
   Q_INVOKABLE void setSelection(bool selected);
 
+  ///@{
+  /// Enabled Servers
+  /// ["All"] by default
```

Comment:
Can we avoid this special `All` value? I would not allow all servers by default anyway (just where the image is retrieved from). It is also quite messy - the string needs to be translated, someone might accidentally use `All` as connection name, etc. (a separate flag or enum would be cleaner, but we can add it in the future, if we find that it is actually necessary to always automatically allow querying patient data from a server that did not originally provide it).